network-api/networkapi/wagtailpages/pagemodels/campaigns.py

```diff
@@ -368,6 +374,11 @@ class BanneredCampaignPage(PrimaryPage):
 
     tags = ClusterTaggableManager(through=BanneredCampaignTag, blank=True)
 
+    aside_page_content = models.BooleanField(
+        default=False,
+        help_text=('Turning this on will create a right hand sidebar on large '
+                   'screens for any aside blocks added to the body.')
+        )
```

Comment:
Unfortunately there is currently no formatter set up, but what do you think about outdenting the closing parenthesis one level? 
```suggestion
    )
```

Reply:
Noted thanks, this switch field has now been removed